src/pages/users/[id]/share.tsx

```diff
@@ -147,60 +161,48 @@ const SharePage = () => {
     setShareTelegram(event.target.checked);
   };
 
+  if (!profile) {
+    return <div>Loading...</div>;
+  }
+
   if (!user) {
     return <div>User not found</div>;
   }
 
   return (
-    <div className="p-4">
-      <h3 className="text-lg leading-6 font-medium text-gray-900 dark:text-white">
-        Connect with {user.name}
-      </h3>
-      <div className="mt-4">
-        <label className="inline-flex items-center">
-          <input
-            type="checkbox"
-            className="form-checkbox"
-            disabled={!user.x}
-            checked={shareTwitter}
-            onChange={handleTwitterChange}
-          />
-          <span className="ml-2">Share my Twitter @{user.x}</span>
-        </label>
-      </div>
-      <div className="mt-4">
-        <label className="inline-flex items-center">
-          <input
-            type="checkbox"
-            className="form-checkbox"
-            disabled={!user.tg}
-            checked={shareTelegram}
-            onChange={handleTelegramChange}
-          />
-          <span className="ml-2">Share my Telegram @{user.tg}</span>
-        </label>
-      </div>
-      <div className="mt-4">
-        <label className="inline-flex items-center">
-          <input
-            type="longtext"
-            disabled
-            value={privateNote}
-            onChange={(event) => {
-              setPrivateNote(event.target.value);
-            }}
-          />
-          <span className="ml-2">Private Note</span>
-        </label>
-      </div>
-      <div className="mt-8">
-        <button
-          className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-          onClick={handleConnectClick}
-        >
-          Connect
-        </button>
-      </div>
+    <div>
+      <AppBackHeader />
+      <FormStepLayout
+        title={`Connect with ${user.name}`}
+        description="Share your contact information with this user."
+        onSubmit={handleConnect}
+      >
+        <Input
+          type="checkbox"
+          className="form-checkbox"
+          label={`Share my Twitter: @${profile.twitterUsername}`}
```

Comment:
Definitely confusing that this alternates betwen `twitterUsername` and `x` for different objects

Reply:
The rule is x is used whenever space matters and twitterUsername is used everywhere else, to be more descriptive.